src/Clerk.ts

```diff
@@ -133,6 +148,29 @@ export default class Clerk {
     return this._userApi;
   }
 
+  // Utilities
+
+  decodeToken(token: string): jwt.JwtPayload {
+    const decoded = jwt.decode(token)
+    if (!decoded) {
+      throw new Error('Failed to decode token')
+    }
+
+    return decoded as jwt.JwtPayload
+  }
+
+  async verifyToken(token: string): Promise<jwt.JwtPayload> {
+    const decoded = jwt.decode(token, { complete: true })
+    if (!decoded) {
+      throw new Error('Failed to decode token')
+    }
+
+    const key = await this._jwksClient.getSigningKey(decoded.header.kid)
+    const verified = jwt.verify(token, key.getPublicKey(), { algorithms: ['RS256'], audience: 'clerk'})
```

Comment:
🙃 Let's make the algorithms array another option that defaults to `['RS256']` to make the code more futureproof.